static/.htaccess

```
@@ -8,6 +8,11 @@ Options -Indexes -MultiViews
 
 # Redirect pages - https://httpd.apache.org/docs/current/mod/mod_rewrite.html#rewriterule
 
+# 8.1 apis/clients to apis/tools change
+RewriteRule ^docs/apis-clients(.*)$ /docs/apis-tools$1 [R=301,L]
+RewriteRule ^optimize/apis-clients(.*)$ /optimize/apis-tools$1 [R=301,L]
+RewriteRule ^docs/apis-clients/working-with-apis-clients/?$ /docs/apis-tools/working-with-apis-tools/ [R=301,L]
```

```suggestion
RewriteRule ^docs/apis-clients/working-with-apis-clients/?$ /docs/apis-tools/working-with-apis-tools/ [R=301,L]
RewriteRule ^docs/apis-clients(.*)$ /docs/apis-tools$1 [R=301,L]
RewriteRule ^optimize/apis-clients(.*)$ /optimize/apis-tools$1 [R=301,L]
```

The current link-checking issue is because the more generic rule is matching, before it can get to the more specific rule! So `/docs/apis-clients/working-with-apis-clients/` is being redirected to `/docs/apis-tools/working-with-apis-clients/` -- the new folder is correct, but the new filename isn't changing. 

If we move the more-specific rule first, then that one will be caught instead.